regparser/tree/reg_text.py

```diff
@@ -83,6 +83,27 @@ def build_subpart(text, part):
     return struct.Node(
         "", [], label, subpart_title, node_type=struct.Node.SUBPART)
 
+def build_subjgrp(text, part):
+    """
+    We're constructing a fake "letter" here by taking the first letter of
+    each word in the subjgrp's title, or using the first two letters of the
+    first word if there's just one—we're avoiding single letters to make sure we
+    don't duplicate an existing subpart, and we're hoping that the initialisms
+    created by this method are unique for this regulation.
+    We can make this more robust by accepting a list of existing initialisms and
+    returning both that list and the Node, and checking against the list as we
+    construct them.
+    """
+    def subjgrp_label(candidate_text):
```

Comment:
The initialism shows up in the contents, though, and looks far better (and more intentional) than it would as a hash, so I prefer this plus checking against existing titles rather than a hash function. I can see the need to make it more robust, however, and am fine with coming up with that approach before merging.